Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp

```diff
@@ -78,6 +78,12 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
     amrex::GpuArray<int, 3> const& My_stag             = macroscopic_properties->My_IndexType;
     amrex::GpuArray<int, 3> const& Mz_stag             = macroscopic_properties->Mz_IndexType;
     amrex::GpuArray<int, 3> const& macro_cr            = macroscopic_properties->macro_cr_ratio;
+    amrex::GpuArray<int, 3> const& Hx_stag             = macroscopic_properties->Hx_IndexType;
+    amrex::GpuArray<int, 3> const& Hy_stag             = macroscopic_properties->Hy_IndexType;
+    amrex::GpuArray<int, 3> const& Hz_stag             = macroscopic_properties->Hz_IndexType;
+    amrex::GpuArray<int, 3> const& Bx_stag             = macroscopic_properties->Bx_IndexType;
+    amrex::GpuArray<int, 3> const& By_stag             = macroscopic_properties->By_IndexType;
+    amrex::GpuArray<int, 3> const& Bz_stag             = macroscopic_properties->Bz_IndexType;
```

Comment:
`H_stag` is the same to `B_stag`, isn't it?

Reply:
Yes they are the same in our case, but to keep the code generic we can go with separate stageerings for B and H updates.